scripts/create_stream_product.ts

```diff
@@ -14,26 +14,27 @@ import CoreAPIClient from "../src/CoreAPIClient"
 
 const { log } = console
 
-import { Chains } from "@streamr/config"
+import { networks } from "@streamr/config"
 
 const {
-    ethereum: {
+    dev0: {
         rpcEndpoints: [{
             url: ETHEREUM_SERVER_URL,
         }],
         contracts: {
-            "Marketplace": MARKETPLACE_ADDRESS,
+            "DATA": TOKEN_ADDRESS,
+            "MarketplaceV3": MARKETPLACE_ADDRESS,
         }
     },
-    streamr: {
+    dev1: {
         rpcEndpoints: [{
             url: MATIC_SERVER_URL,
         }],
         contracts: {
             "StreamRegistry": STREAM_REGISTRY_ADDRESS,
         }
     }
-} = Chains.load("development")
+} = networks
```

Comment:
not sure what you mean. VSCode/tsc can still deduce correct type for the imported items (`string` for all...) and also 
 it complains if I try to import something that doesn't exist. So at least in that sense all the type safety you could want in this case is there.

Reply:
So if you use `Chains.load()` you will get correct type information from config package. Why use Typescript at all if type information is useless in the first place?

Reply:
The type information is there even when you simply get the "network". Try it!

Reply:
E.g. if you try
```
        contracts: {
            "DATA-token": TOKEN_ADDRESS,
        }
```
instead of the correct
```
        contracts: {
            DATA: TOKEN_ADDRESS,
        }
```
you will get a compile error: 
```
Property 'DATA-token' does not exist on type '{ AMB: string; DATA: string; ENS: string; FIFSRegistrar: string; Marketplace: string; MarketplaceV3: string; MigrationManager: string; PublicResolver: string; TokenMediator: string; TrackerRegistry: string; ... 4 more ...; "core-api": string; }'.ts(2339)
```

Reply:
When I say types I refer to the class structure returned from `Chains.load()`.

Reply:
Well that isn't needed here. Those types aren't used anywhere. What is completely sufficient is that the type system prevents typos / outdated names. This is something Javascript doesn't provide but Typescript does, which is great.